### PR TITLE
Max path bytes

### PR DESCRIPTION
**Pull request by sebn**

### CODE REVIEW

**Comment by sebn** [comment]
- [x] Fix linux build
- [ ] Fix macOS build
- [ ] Fix windows build

**Comment by sebn** [comment]
Windows tests pass locally. AppVeyor fails because of some missing cozy instance (not related to this PR). I'll try to fix it later.

File: gui/app/locales/fr.json [hunk 1]
```diff
@@ -42,6 +42,9 @@
   "Error {0} names cannot end with character {1}": "un nom de {0} ne peut pas se terminer par un caractère {1}",
   "Error {0} names cannot include characters {1}": "un nom de {0} ne peut pas inclure de caractères {1}",
   "Error the “{0}” name is reserved": "le nom « {0} » est réservé",
+  "Error it exceeds the path size limit": "il dépasse la limite de taille pour les chemins",
+  "Error it exceeds the name size limit": "il dépasse la limite de taille pour les noms",
+  "Error it exceeds the folder name size limit": "il dépasse la limite de taille pour les noms de dossiers",
```
Comment (nono) [inline gui/app/locales/fr.json]: These lines should be added to `gui/app/locales/en.json` too, no? (and `tx push -s` to make them available to translate in transifex)

Comment (sebn) [inline gui/app/locales/fr.json]: Agreed

Comment (sebn) [inline gui/app/locales/fr.json]: Fixed

**Comment by sebn** [comment]
Code could be improved, but I believe it's ok for the upcoming release. WDYT?

**Pull request merged**